torch/backends/_nnapi/serializer.py

```diff
@@ -1334,7 +1334,7 @@ def add_pointwise_simple_unary_op(self, node, opcode):
         self.add_operation(opcode, inputs, outputs)
 
     def _do_add_binary(self, node, opcode, fuse_code, *, qparams=None):
-        """Helper for pointwise binary broadcast ops with superfluous extra args"""
+        """Assist with Pointwise binary broadcast ops with superfluous extra args."""
```

Comment:
Not sure about this. "Helper" is a common name for such functions, "assist" sounds very unnatural.

Reply:
I understand, but pydocstyle doesn't like "Helper" as a first word as this is not "imperative". Do you have other ideas on how I can start this docstring.

Reply:
@nvs-abhilash Maybe add an ignore comment for this one? The function is private, starts with underscore, so not user-facing.